dkc/core/rest/file.py

```diff
@@ -115,6 +117,23 @@ def perform_create(self, serializer: FileSerializer):
                 {'size': ['This file would exceed the size quota for this folder.']}
             )
 
+    def perform_update(self, serializer: FileUpdateSerializer) -> None:
+        with transaction.atomic():
```

Comment:
Is it always necessary to open a transaction? Can it only be done if this update contains `'blob'`?